doc/technotes/localeModels.rst

```diff
@@ -96,8 +97,8 @@ Qthreads thread scheduling
 
 When qthreads tasking is used, different Qthreads thread schedulers are
 selected depending upon the ``CHPL_LOCALE_MODEL`` setting.  For the flat
-locale model the "nemesis" thread scheduler is used, and for the NUMA
-locale model the "distrib" thread scheduler is used.  This selection is
+locale model the "nemesis" thread scheduler is used, and for the NUMA and KNL
+locale models the "distrib" thread scheduler is used.  This selection is
 done at the time the Qthreads third-party package is built, and cannot
 be adjusted later, either at user compile time or at execution time.
 
```

Comment:
Seems like we should add a section for the KNL locale model to this file. Doesn't have to be done at part of this commit, but I think we wants something in by doc freeze.